frontend/src/assets/svg/Svg.js

```diff
@@ -0,0 +1,109 @@
+const Svg = ({ icon }) => {
+  switch (icon) {
+    case "moon":
+      return (
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="20"
+          height="20"
+          fill="currentColor"
+          className="bi bi-moon-stars-fill"
+          viewBox="0 0 16 16"
+        >
+          <path d="M6 .278a.768.768 0 0 1 .08.858 7.208 7.208 0 0 0-.878 3.46c0 4.021 3.278 7.277 7.318 7.277.527 0 1.04-.055 1.533-.16a.787.787 0 0 1 .81.316.733.733 0 0 1-.031.893A8.349 8.349 0 0 1 8.344 16C3.734 16 0 12.286 0 7.71 0 4.266 2.114 1.312 5.124.06A.752.752 0 0 1 6 .278z" />
```

Comment:
It looks better separated like that, but I had something like that in mind: https://www.npmjs.com/package/react-bootstrap-icons. Let me know what you think about this approach

I only wonder why it's not part of official react-bootstrap package

Reply:
Yeah ideally this whole file should be replaced by that I hope.

Reply:
Sure, I'll do that!

Reply:
Done